railties/lib/rails/generators/rails/app/templates/app/assets/javascripts/cable.js

```diff
@@ -1,5 +1,5 @@
 // Action Cable provides the framework to deal with WebSockets in Rails.
-// You can generate new channels where WebSocket features live using the rails generate channel command.
+// You can generate new channels where WebSocket features live using the `rails generate channel` command.
```

Comment:
Changes to template comments should not be ci-skipped. Can you remove that from commit?